impress-org/givewp
# Feature: Form inherits campaign goal

## Description

WIP

This PR adds functionality that sets the form goal based on the campaign goal. 

## Affects

Form goal


## Testing Instructions

Create a new campaign
Set campaign goal
Verify that the default campaign form has the same goal settings as the campaign
Create another form and verify that the new form also has the same goal settings as the campaign. 



## Pre-review Checklist
